conf/jetty/jetty.xml.production

```diff
@@ -754,7 +754,7 @@
     %%comment SERVICE:zimbra,<!--%% ZIMBRAWEBAPPEND -->
 
     <!-- ZIMBRAADMINWEBAPPBEGIN %%comment SERVICE:zimbraAdmin,-->%%
-    <New id="zimbraAdmin" class="org.eclipse.jetty.webapp.WebAppContext">
+    <!--<New id="zimbraAdmin" class="org.eclipse.jetty.webapp.WebAppContext">
```

Comment:
If this part is commented it makes sense to remove it. Is git afterall, we can check what it was in the past commits

Reply:
@Polpetta makes sense, the idea is to un-comment it in production server if the sysadmin wants to re-enable the webapp :)
This is one of the core requirement of the ticket.

Reply:
okok!